Adds args with string fallback to aria-labels in components (#1373)
hashicorp/design-system: website/docs/components/breadcrumb/partials/code/component-api.md

### Breadcrumb::Truncation

<Doc::ComponentApi as |C|>

>   <C.Property @name="ariaLabel" @type="string">
>     Set on the truncation toggle button. Accepts a localized string; the fallback is set to `show more`.
>   </C.Property>

Reviewer: ```suggestion
  <C.Property @name="ariaLabel" @type="string" @default="show more">
    Set on the truncation toggle button. Accepts a localized string.
  </C.Property>
```
